core/css.js

```diff
@@ -934,6 +934,9 @@ Blockly.Css.CONTENT = [
   '.blocklyDropDownDiv .goog-option-selected .goog-menuitem-icon {',
      /* Client apps may override the URL at which they serve the sprite. */
     'background: url(<<<PATH>>>/sprites.png) no-repeat -48px -16px !important;',
+    'position: static;', /* Scroll with the menu. */
+    'float: left;',
```

Comment:
yeah..hmmm..maybe closure itself should fix the issue where the checkbox doesn't scroll, I guess. but I can try to think of some rtl hack or something